src/components/AccountDetails/Copy.tsx

```diff
@@ -6,23 +6,38 @@ import { LinkStyledButton } from '../../theme'
 import { CheckCircle, Copy } from 'react-feather'
 
 const CopyIcon = styled(LinkStyledButton)`
-  color: ${({ theme }) => theme.text3};
+  color: ${({ theme }) => theme.text4};
   flex-shrink: 0;
   display: flex;
   text-decoration: none;
   font-size: 0.825rem;
+  padding: 0;
   :hover,
   :active,
   :focus {
     text-decoration: none;
     color: ${({ theme }) => theme.text2};
   }
 `
+
+const TransactionStatus = styled.span`
+  ${({ theme }) => theme.flexRowNoWrap};
+  align-items: center;
+  align-items: center;
+`
+
 const TransactionStatusText = styled.span`
+  color: ${({ theme }) => theme.text4};
   margin-left: 0.25rem;
   font-size: 0.825rem;
-  ${({ theme }) => theme.flexRowNoWrap};
-  align-items: center;
+`
+
+const CustomCopy = styled(Copy)`
```

Comment:
I think it might be a good thing to see where the `Copy` component is used elsewhere and see if it has problems with the new color. Because I think the idea is to have purple stuff pretty much everywhere.

Reply:
@luzzif  the Copy component is only used here